src/main.rs

```diff
@@ -586,7 +662,7 @@ fn main_recv(
             // end of the channel, just crash the entire program so that the
             // error message is clearer.
             file.handle_chunk(chunk).expect("Failed to write chunk.");
-            print_progress(bytes_received, total_len, start_time);
+            print_progress(bytes_received, total_len, start_time).ok();
```

Comment:
The `.ok()` doesn’t do anything except silently ignore the result of `print_progress`. If that’s the goal,

```suggestion
            let _ = print_progress(bytes_received, total_len, start_time);
```

is a more explicit way of expressing that intent. If you want to return from the function in case of error, we can write

```suggestion
            print_progress(bytes_received, total_len, start_time)?;
```

Or if you meant to assert that the result is `Ok`, and want to crash the program if it’s not, then we can do

```suggestion
            print_progress(bytes_received, total_len, start_time).expect("We should be able to print to stdout.");
```

One cases where printing to stdout can fail, is when you pipe the output to a different program that then exits before this one, leaving a pipe that you can’t write to any more.